grails-app/views/aboutpage.gsp

```diff
@@ -1,130 +1,94 @@
-                                <img src='${resource(dir: "images/", file: "jenn.jpg")}' alt="Thumbnail Image" class="img-circle img-responsive">
-                                <h5>Jenn Strater<br /><i class="fa fa-twitter"><small class="text-muted"><a href="https://twitter.com/codeJENNerator">@codeJENNerator</i></a></small></h5>
-                                <p>Jenn is a Developer Advocate at Gradle. Also the owner of the Groovy Community Slack;Contributor to Spring REST Docs,Codernarc and more.</p>
+                                <img src='${resource(dir: "images/", file: "placeholder.jpg")}' alt="Thumbnail Image" class="img-circle img-responsive">
+                                <h5>Jenn Strater<br /><small class="text-muted">@twitter</small></h5>
+                                <p>You can write here details about one of your team members. You can give more details about what they do. Feel free to add some <a href="#">links</a> for people to be able to follow them outside the site.</p>
                             </div>
                         </div>
                         <div class="col-md-4">
                             <div class="team-player">
-                                <img src='${resource(dir: "images/", file: "ken.jpg")}' class="img-circle img-responsive">
-                                <h5>Ken Kousen<br /><small class="text-muted"><i class="fa fa-twitter"><a href="https://twitter.com/kenkousen">@kenkousen</i></a></small></h5>
-                                <p>Ken Kousen is an independent consultant and trainer specializing in Spring,Hibernate,Groovy and Grails. He holds numerous technical certifications, along with degrees in Mathematics and Aerospace Engineering, and Computer Science.<a href="https://twitter.com/kraulain"></a></p>
+                                <img src='${resource(dir: "images/", file: "placeholder.jpg")}' class="img-circle img-responsive">
+                                <h5>Ken Kousen<br /><small class="text-muted">@twitter</small></h5>
+                                <p>You can write here details about one of your team members. You can give more details about what they do. Feel free to add some <a href="#">links</a> for people to be able to follow them outside the site.</p>
                             </div>
                         </div>
                         <div class="col-md-4">
                             <div class="team-player">
                                 <img src='${resource(dir: "images/", file: "placeholder.jpg")}' class="img-circle img-circle img-responsive">
                                 <h5>Paul King<br /><small class="text-muted">@twitter</small></h5>
                                 <p>You can write here details about one of your team members. You can give more details about what they do. Feel free to add some <a href="#">links</a> for people to be able to follow them outside the site.</p>
+                                <footer>
```

Comment:
footer doesn't look like that in the template. (or that on home page)
please fix the footer so that it can display properly